### DISCUSSION TITLE
How to get systemd to umount mounts before stopping netbird

### DISCUSSION THREAD

**Discussion by gene1wood** [netbirdio/netbird#570] [Q&A / Support] [answered]
I mount an NFS export over Netbird on my Ubuntu client. What's the best way to get systemd to `umount` those NFS shares on the client **before** stopping Netbird? Currently shutdown always takes a long time as systemd stops Netbird before unmounting the shares and so they have the unmount has to timeout before the machine shuts down.

**Comment by mlsmaycon** [comment, marked as answer]
hello @gene1wood, you can modify the systemd file,`/etc/systemd/system/netbird.service`, for the netbird service and add a `ExecStop=` command to umount the exports. 

Once you set this up, you need to run systemctl daemon-reload to ensure it takes effect.
